src/scripts/views/finances.js

```diff
@@ -57,5 +60,10 @@ var Marionette = require("backbone.marionette");
       },
       checkMoreButton: function () {
           this.$(".more").toggle(this.collection.more);
+      },
+      serializeData: function () {
+          return _.extend({
```

Comment:
`_.extend()` takes two params. (1) the additional data, and (2) the original object you're extending. See other views for examples.